packages/wallet-specs/package.json

```diff
@@ -24,6 +34,17 @@
     "tslint": "5.20.0",
     "tslint-config-prettier": "1.18.0",
     "tslint-plugin-prettier": "2.0.1",
-    "typescript": "^3.7.2"
+    "typescript": "^3.7.2",
+    "@typescript-eslint/eslint-plugin": "^2.7.0",
+    "@typescript-eslint/parser": "^2.8.0",
+    "eslint": "^6.6.0",
+    "eslint-config-prettier": "^6.7.0",
```

Comment:
Should these be dev dependencies of the root repo? The root `eslintrc.js` uses them ...

Reply:
Probably, yes